File: app/src/main/java/io/github/ovso/righttoknow/framework/adapter/BaseViewHolder.java
package io.github.ovso.righttoknow.framework.adapter;

import android.view.View;
import androidx.recyclerview.widget.RecyclerView;
import butterknife.ButterKnife;

/**
 * Created by jaeho on 2017. 8. 1
 */

public class BaseViewHolder extends RecyclerView.ViewHolder {
  public BaseViewHolder(View itemView) {
    super(itemView);
    ButterKnife.bind(this, itemView);
  }
}

File: app/src/main/java/io/github/ovso/righttoknow/framework/adapter/BaseViewHolder.kt
package io.github.ovso.righttoknow.framework.adapter

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import butterknife.ButterKnife

class BaseViewHolder(itemView: View?) : RecyclerView.ViewHolder(itemView!!) {
  init {
    ButterKnife.bind(this, itemView!!)
  }
}